.eslintrc

```diff
@@ -1,6 +1,11 @@
 {
-  "extends": ["eslint:recommended", "plugin:react/recommended"],
-  "parser": "@babel/eslint-parser",
+  "root": true,
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended"
```

Comment:
we should also add `plugin:@typescript-eslint/eslint-recommended` here as it effectively disables the conflicting eslint rules and replaces them with typescript aware implementations

Reply:
also, do you maybe have warnings and error numbers with `plugin:@typescript-eslint/recommended` extended just to have a rough idea of what is needed to enforce a more standard configuration

Reply:
without the .ts,.tsx overrides we get:
✖ 353 problems (40 errors, 313 warnings)